### ISSUE TITLE
[A11y] - Data table has missing or incomplete header cell markup

### ISSUE THREAD

**Issue by VaniChinnam** [dequelabs/cauldron#1390]
### Current Behaviour 
`The data table has missing or incomplete header cell markup. `<br/><hr/>
### Expected Behaviour 
`RULE :  Table headers MUST be designated with <th>. HOW TO FIX:  Fix this issue by doing ALL of the following: 1. Mark all column header and/or row header cells with a <th> element. 2. Ensure that header cell and data cell relationships are correctly conveyed according to the complexity of the table. `<br/><hr/>
### Impact 
 <p>

- [ ] Blocker
- [x] Serious
- [ ] Critical
- [ ] Moderate
- [ ] Minor

### Issue Details

<details><summary>View Issue 1685576 </summary> 
 <p>

_**Auditor ID**_ : [1685576](https://axeauditor.dequecloud.com/issue-global-search?searchKeyword=1685576&searchOn=id) 

_**description**_ :  The "data" present under the "Name" column should be marked as <th> 
Element location: In all the screens wherever we have tables. 
e.g.  -Spacing -Typography -Accordion -Address -Alert -Breadcrumb -Button -Checkbox -Click Outside listener and so on... 

_**Page Name**_ : Project Wide 

_**source**_ : `<td class="TableCell">Base</td>` 

_**Screenshots**_ : [Table headers](https://axeauditor.dequecloud.com/api/v1/file/0115f732-daea-11ee-8bee-7f1a8aa2bb66
Table - headers: https://axeauditor.dequecloud.com/api/v1/file/aa83dca4-dadf-11ee-b520-1f35a7127684) 

</p>
</details>

**Comment by scurker** [comment]
@VaniChinnam for my understanding, is there a reason this is being raised as an issue? We discussed this on the team internally and we strongly feel that none of those columns should be row headers. The way we approached this is that it's contextually dependent on the value the user is looking for and not any particular column should be the header.

**Comment by VaniChinnam** [comment]
@scurker Agree that it depends on the value present in tables. Hence closing this issue.
